lib/mbta_metro/components/accordion.ex

```diff
@@ -38,7 +38,7 @@ defmodule MbtaMetro.Components.Accordion do
   def accordion(%{variant: "contained"} = assigns) do
     ~H"""
     <details id={@id} class="group w-full bg-white" open={@open}>
-      <summary class="border border-solid border-blue-500 text-blue-600 hover:text-slate-900 hover:bg-blue-100 p-2 flex cursor-pointer list-none items-center gap-2 p-2 relative">
+      <summary class="border border-solid border-blue-500 text-blue-600 hover:text-slate-900 hover:bg-blue-100 p-2 flex cursor-pointer list-none items-center gap-2 relative">
```

Comment:
Lol it's important to _really_ make sure that it's got 8px of padding 😂

Reply:
One of the things I don't like about Tailwind...nearly impossible to see what styles are being applied once you have more than a handful.

Reply:
Yeah there's a pattern I've seen elsewhere in our code where we group the styles approximately by what they represent (layout, color, etc, for instance), and break them up into separate lines. It's a shame that there's no way to enforce or encourage it at the CI level though, because at the Elixir-level, it's just strings.

Reply:
There is a VSCode plugin that sorts classes according to the official sorting. I prefer alphabetical so you can actually read through it. But, I don't think we have enough styles to make it really matter.